doc/index.md

```diff
@@ -19,32 +19,14 @@
 
 Spglib is a library for finding and handling crystal symmetries
 written in C. There are the {ref}`python <python_spglib>`,
-{ref}`fortran, rust, and ruby <interfaces>` interfaces.
```

Comment:
@atztogo So, the Ruby interface is removed. Is it OK to be done in minor version changes?

Reply:
Oh, fair point. Can we confirm that the ruby interface was working in  2.0.2. If not then yes we could, otherwise we should add deprecation notices.

Reply:
@atztogo Can you comment on the ruby interface and what we should do until maybe `v2.2`? If it is still functioning, we can mark it deprecated, but if not, we might need to consider the course of actions.

Reply:
For those that I could manage, the wrappers have been provided from spglib repository. Those not had been maintained as an independent project. But we may have to consider it for the future.

I think we can drop the current ruby support. We may say "but contribution is welcome." We also think about the rust support (https://github.com/spglib/spglib-rs), which, I think, doesn't follow the change of spglib-v2. 
@seatonullberg, how do we do? It seems there is a Julia wrapper, https://github.com/singularitti/Spglib.jl, too. Some will may develop for go-lang, etc.